content/management/configuration-api/v3.3/index.mdx

```diff
@@ -3600,13 +3607,15 @@ curl -X POST \
 |                 |                                                                            |
 | --------------- | -------------------------------------------------------------------------- |
 | Method URL      | `https://api.livechatinc.com/v3.3/configuration/action/unregister_webhook` |
-| Required scopes | `webhooks--my:rw` `webhooks--all:rw`                                       |
+| Required scopes | `webhooks.configuration:rw`                                       |
 
 #### Request
 
-| Parameter | Required | Data type | Notes      |
-| --------- | -------- | --------- | ---------- |
-| `id`      | Yes      | `string`  | Webhook ID |
+| Parameter         | Required | Data type | Notes                                                                                        |
+| ----------------- | -------- | --------- | -------------------------------------------------------------------------------------------- |
+| `id`              | Yes      | `string`  | Webhook ID                                                                                   |
+| `owner_client_id` | no       | `string`  | ClientID of desired webhook owner. Required when Personal Access Token is used to authorize. |
```

Comment:
```suggestion
| `owner_client_id` | no       | `string`  | The webhook owner (the Client Id for which the webhook is registered). It's required when a Personal Access Token is used for authorization. |
```

Reply:
Method definition proposal for [**Unregister Webhook**](https://github.com/livechat/livechat-public-docs/pull/814/files#diff-8ad53a5eba0e6c361340079931ab39577b913fa57076b7150d8d07cb95da1fb2R3603):

Unregisters a webhook previously [registered](#register-webhook) for a Client Id (application).